app/models/application_record.rb

```diff
@@ -1,3 +1,7 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+
+  def tpying
+    puts 'esential typo'
```

Comment:
__[staging][RuboCop]__

Rails/Output: Do not write to stdout. Use Rails's logger if you want to log.


<!-- Comment by [Sider](https://sider.review) -->